test/Benchmarks/Helper/LocalServer.cs

```diff
@@ -0,0 +1,92 @@
+﻿// <copyright file="LocalServer.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NETCOREAPP
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
+namespace Benchmarks.Helper
+{
+    public class LocalServer : IDisposable
+    {
+        private const string SourceName = "http-client-test";
```

Comment:
Not used, right?

Reply:
It's used by AddSource. Should I remove AddSource?

Reply:
I mean - none is producing activity from this source. So you can remove both AddSource and SourceName.

Reply:
done!